src/hooks/game/validateBalanceGameForm.ts

```diff
@@ -24,9 +25,17 @@ export const validateBalanceGameForm = (
 };
 
 export const validateGameTag = (form: BalanceGame) => {
+  const subTagList = createArrayFromCommaString(form.subTag);
+
   if (isEmptyString(form.mainTag)) {
     return { isValid: false };
   }
+  if (subTagList.length > 3) {
+    return { isValid: false };
+  }
+  if (!isAllLessThan(subTagList, 10)) {
+    return { isValid: false };
+  }

```

Comment:
```suggestion
 const subTagList = createArrayFromCommaString(form.subTag);
  const isValid =
    !isEmptyString(form.mainTag) &&
    subTagList.length <= 3 &&
    isAllLessThan(subTagList, 10);
  return { isValid };
```
로 일괄 처리 하는 방식도 좋아 보이네요!

Reply:
[e7c0e2f](https://github.com/JECT-Study/PICK-O-Client/pull/306/commits/e7c0e2fa2b298e4e62d18f8b4f34190b233840a1) 에서 수정했습니다!!